test/runner.sh

```diff
@@ -46,19 +46,9 @@ TEST_ROLE_READ_ONLY=${TEST_ROLE_READ_ONLY:-test_role_read_only}
 shift
 
 # Drop test database and make it less verbose in case of dropping a
-# distributed database. Also revoke privileges granted when setting up
-# the template1 database. This has to be revoked since the user is
-# dropped from a different database.
+# distributed database.
 function cleanup {
-    ${PSQL} "$@" -U ${USER} -d template1 -v ECHO=none >/dev/null 2>&1 <<EOF
-REVOKE CREATE ON SCHEMA public FROM ${TEST_PGUSER};
-REVOKE CREATE ON SCHEMA public FROM ${TEST_ROLE_DEFAULT_PERM_USER};
-REVOKE CREATE ON SCHEMA public FROM ${TEST_ROLE_DEFAULT_PERM_USER_2};
-REVOKE CREATE ON SCHEMA public FROM ${TEST_ROLE_1};
-REVOKE CREATE ON SCHEMA public FROM ${TEST_ROLE_2};
-REVOKE CREATE ON SCHEMA public FROM ${TEST_ROLE_3};
-EOF
```

Comment:
I think it is necessary to remove the privileges before dropping the users, which caused failures in appveyor. If it passes, I'm fine.